chainer/backend.py

```diff
@@ -70,6 +71,23 @@ def copyto(dst, src):
             type(dst)))
 
 
+def get_default_device_from_array_module(xp):
```

Comment:
I think it's better to keep it private.
The term "default device" is not clearly defined. CUDA's current device is not "the default device".
There's also a discussion to introduce the concept of Chainer's "default device", which is different from this one.